publications-2020.bib

```diff
@@ -657,6 +657,17 @@ @Article{2020:fischer.min.ea:scalability
   publisher = {{SAGE} Publications}
 }
 
+@Article{2020:freese:numerical,
+  author  = {Freese, Jan Philip},
+  title   = {Numerical homogenization of time-dependent Maxwell's equations with dispersion effects},
+  year    = 2020,
```

Comment:
The exam was in 2020, but the manuscript was published 2021. To which year do we assign these theses?